crates/driver/src/infra/api/solve/dto/auction.rs

```diff
@@ -169,7 +170,9 @@ struct Order {
     #[serde_as(as = "serialize::U256")]
     executed: eth::U256,
     interactions: Vec<Interaction>,
+    #[serde(default)]
     sell_token_balance: SellTokenBalance,
+    #[serde(default)]
     buy_token_balance: BuyTokenBalance,
```

Comment:
Should these rather be optional?
Otherwise we might encounter an error while fetching the balances and a solver trusting those fields would assume this order can not be settled.

Reply:
I'm specifically thinking about [this](https://cowservices.slack.com/archives/C036JAGRQ04/p1673260079140729) idea. If we decide to go the route of letting solvers decide which order to include based on `sell_token_balance` it would matter for them if the value is actually 0 or if we just encountered an error while computing the amount and defaulted to 0.
If, on the other hand, we make the driver responsible for querying the current balances for each order we might as well drop those fields from the order coming from the `autopilot` altogether.